Slurm job started at 15/06/2023_21:00:34

NNODES = 6
SIMUNODES = 4
WORKERNODES = 2
NPROC = 128
NWORKER = 60
......['tcp://10.7.0.142:33063', 'tcp://10.7.0.142:33755', 'tcp://10.7.0.142:34767', 'tcp://10.7.0.142:34871', 'tcp://10.7.0.142:38069', 'tcp://10.7.0.142:38247', 'tcp://10.7.0.142:38396', 'tcp://10.7.0.142:38993', 'tcp://10.7.0.142:38997', 'tcp://10.7.0.142:39237', 'tcp://10.7.0.142:39256', 'tcp://10.7.0.142:39528', 'tcp://10.7.0.142:39894', 'tcp://10.7.0.142:40158', 'tcp://10.7.0.142:40567', 'tcp://10.7.0.142:40575', 'tcp://10.7.0.142:40666', 'tcp://10.7.0.142:41289', 'tcp://10.7.0.142:41907', 'tcp://10.7.0.142:42182', 'tcp://10.7.0.142:42455', 'tcp://10.7.0.142:42528', 'tcp://10.7.0.142:42557', 'tcp://10.7.0.142:42702', 'tcp://10.7.0.142:43221', 'tcp://10.7.0.142:44919', 'tcp://10.7.0.142:45683', 'tcp://10.7.0.142:45998', 'tcp://10.7.0.142:46174', 'tcp://10.7.0.142:46212', 'tcp://10.7.0.144:32970', 'tcp://10.7.0.144:33158', 'tcp://10.7.0.144:33222', 'tcp://10.7.0.144:33755', 'tcp://10.7.0.144:34433', 'tcp://10.7.0.144:34745', 'tcp://10.7.0.144:35584', 'tcp://10.7.0.144:36464', 'tcp://10.7.0.144:36488', 'tcp://10.7.0.144:37261', 'tcp://10.7.0.144:37512', 'tcp://10.7.0.144:37530', 'tcp://10.7.0.144:37684', 'tcp://10.7.0.144:39011', 'tcp://10.7.0.144:39845', 'tcp://10.7.0.144:40627', 'tcp://10.7.0.144:40671', 'tcp://10.7.0.144:40864', 'tcp://10.7.0.144:41266', 'tcp://10.7.0.144:41422', 'tcp://10.7.0.144:41601', 'tcp://10.7.0.144:41686', 'tcp://10.7.0.144:42547', 'tcp://10.7.0.144:42743', 'tcp://10.7.0.144:42983', 'tcp://10.7.0.144:43088', 'tcp://10.7.0.144:43755', 'tcp://10.7.0.144:45342', 'tcp://10.7.0.144:45545', 'tcp://10.7.0.144:46623']
dask.array<concatenate, shape=(12, 2048, 8192), dtype=float64, chunksize=(1, 256, 512), chunktype=numpy.ndarray>
Generated contract {'global_t': [(0, 12, 1)]}
Contract has been signed
Original arrays deleted
SIMULATION_TIME:      35.166316376999994 (avg: 2.930526364750000)
PDI_DELAY:      33.990091865000004 (avg: 2.832507655416667)
SIM_WTHOUT_PDI:       1.176224511999990 (avg: 0.098018709333333)
SUCCESS
EST_ANALYTICS_TIME:            30.36652
Done
tornado.application - ERROR - Exception in callback <bound method Client._heartbeat of <Client: 'tcp://10.7.0.141:8786' processes=60 threads=2400, memory=9.38 TiB>>
Traceback (most recent call last):
  File "/gpfs/users/fernandezx/spack/opt/spack/linux-centos7-cascadelake/gcc-11.2.0/py-tornado-6.2-lemkpcoofzciypjtwijrfhbso5bboah2/lib/python3.10/site-packages/tornado/ioloop.py", line 921, in _run
    val = self.callback()
  File "/gpfs/users/fernandezx/spack/opt/spack/linux-centos7-cascadelake/gcc-11.2.0/py-distributed-deisa-tvck22hdjddjrkrb4clzsednvt2igzro/lib/python3.10/site-packages/distributed/client.py", line 1099, in _heartbeat
    self.scheduler_comm.send({"op": "heartbeat-client"})
  File "/gpfs/users/fernandezx/spack/opt/spack/linux-centos7-cascadelake/gcc-11.2.0/py-distributed-deisa-tvck22hdjddjrkrb4clzsednvt2igzro/lib/python3.10/site-packages/distributed/batched.py", line 136, in send
    raise CommClosedError(f"Comm {self.comm!r} already closed.")
distributed.comm.core.CommClosedError: Comm <TCP (closed) Client->Scheduler local=tcp://10.7.0.141:60760 remote=tcp://10.7.0.141:8786> already closed.

functions called:
HTML_ANALYTICS_TIME:      28.14
HTML_TRANSFER_TIME:       6.03
HTML_DESERIALIZE_TIME:    370.21
HTML_COMPUTE_TIME:        9.81
